File: Data Structures & Algorithms/duplicate-integer/submission-2.cpp
class Solution {
public:
    int b;
    bool hasDuplicate(vector<int>& nums) {
    //  std::vector<int>brr(nums.size(),false);
     
    //  for(int i= 0;i<nums.size();i++){
    //     b = nums[i];
    //     if(brr[b]==false){
    //         brr[b]=true;
    //     }
    //     else{
    //       return true;
    //     }
       
    // }
    // return false;
   std::sort(nums.begin(),nums.end());
   for(int i =0;i<nums.size();i++){
    if(nums[i]==nums[i+1])
    return true;
   }
   return false;
  }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        count = 0
        nums_set = set(nums)
        max_count =0
       ##std::vector<int>maxCount;
        for num in nums_set:
            if num-1 not in nums_set:
              current_num= num
              current_count=1

              while current_num + 1 in nums_set:
                 current_num+=1
                 current_count+=1

              max_count=max(max_count, current_count)
        return max_count
